docs/proposals/20220825-azuremanaged-cluster-v1.md

```diff
@@ -0,0 +1,67 @@
+---
+title: AzureManagedCluster graduation to v1 from experimental
```

Comment:
Does "v1" in this context refer to the literal `apiVersion` or more like "stable" in the sense that the current non-experimental v1beta1 APIs are stable?